### PR TITLE
Updated some text and the styling

### PR DESCRIPTION
**Pull request by jonathan-innis**

### CODE REVIEW

File: ouroboros/hacker/models.py [hunk 1]
```diff
@@ -34,6 +39,17 @@
...
 )
 
+RACES = (
+    ("American Indian", "American Indian or Alaskan Native"),
+    ("Asian", "Asian"),
+    ("Black", "Black or African-American"),
+    ("Hispanic", "Hispanic or Latino White"),
+    ("Native Hawaiian", "Native Hawaiian or other Pacific Islander"),
+    ("White", "White or Caucasian"),
+    ("Two Races", "Two or more races"),
```
Comment (SaltyQuetzals) [inline ouroboros/hacker/models.py]:
```suggestion

```

Comment (SaltyQuetzals) [reply ouroboros/hacker/models.py]: This option is redundant is if we use a `MultiSelectField`

**Comment by SaltyQuetzals** [comment]
@jonathan-innis Please [indicate whether this PR fixes any existing issues in your PR description](https://help.github.com/en/articles/closing-issues-using-keywords).

**Comment by jonathan-innis** [comment]
Does not address any issues currently made

**Comment by SaltyQuetzals** [comment]
@jonathan-innis Please squash before you merge

**Pull request merged**